src/api/app/models/event/repo_publish_state.rb

```diff
@@ -3,6 +3,10 @@ class RepoPublishState < Base
     self.message_bus_routing_key = 'repo.publish_state'
     self.description = 'Publish state of repository changed'
     payload_keys :project, :repo, :state
+
+    def subject
+      "Repository #{payload[:repo]} state of #{payload[:project]} changed to #{payload[:project]}"
```

Comment:
```suggestion
      "Repository #{payload[:repo]} state of #{payload[:project]} changed to #{payload[:state]}"
```
I guess this was a typo.